### PR TITLE
Farm/reload

### PR DESCRIPTION
**Pull request by fakoredeDamilola**
Worked on reloading the app if there is any change
Allow multiple staking and unstaking
https://www.loom.com/share/a1dc20db00c64ebeaa5511ef71e31ff3

### CODE REVIEW

File: app/components/sendToken/index.js [hunk 1]
```diff
@@ -569,7 +569,7 @@ async function updateSendAmount(path, selectedToken, selectedToToken, askAmount,
   if (typeof path[1] != 'undefined') {
     const fromPath = selectedToken.address;
     const toPath = selectedToToken.address;
-    try {
+    // try {
```
Comment (kayroy247) [inline app/components/sendToken/index.js:572]: Removing this try...catch may result to unhandled promise rejection.

File: app/containers/FarmingPage/index.js [hunk 2]
```diff
@@ -47,16 +47,33 @@ export function FarmingPage(props) {
...
+        const minFarmingFee = await masterChef.farmingFee();
+        const fee = Web3.utils.fromWei(minFarmingFee.toString());
+        setFarmingFee(fee);
+        props.changeRGPFarmingFee({
+          fee,
+        });
+      }
+    };
+    RGPfarmingFee();
+    initialLoad ? setFarmingModal(true):setFarmingModal(false)
```
Comment (kayroy247) [inline app/containers/FarmingPage/index.js:69]: Kindly put this in a function and call the function.

**Review by kayroy247 — COMMENTED** [review]
Kindly implement the feedback.

File: app/containers/FarmingPage/index.js (hunk 2 above)
Comment (fakoredeDamilola) [reply app/containers/FarmingPage/index.js]: resolved

**Pull request merged**